lilac2/packages.py

```diff
@@ -48,7 +49,10 @@ def resolve(self) -> Optional[Path]:
       return None
 
   def managed(self) -> bool:
-    return (self.pkgdir / 'lilac.py').exists()
+    conf = read_lilac_yaml(self.pkgdir)
+    if not conf.keys(): # lilac.yaml is missing
+      return False
+    return getattr(conf, 'managed', True)
```

Comment:
We can move `.managed` to `Repo` and store `mods` there.

Reply:
Sounds like a non-trivial refactor. Let me check it out these days.